Projects/Features/SearchFeature/Sources/ViewControllers/SearchViewController.swift

```diff
@@ -19,58 +19,50 @@ public final class SearchViewController: BaseViewController, ViewControllerFromS
     @IBOutlet public weak var contentView: UIView!
     @IBOutlet weak var contentViewBottomConstraint: NSLayoutConstraint!
 
-    var viewModel: SearchViewModel!
-    let disposeBag = DisposeBag()
     var beforeSearchComponent: BeforeSearchComponent!
     var afterSearchComponent: AfterSearchComponent!
     var textPopUpFactory: TextPopUpFactory!
 
-    lazy var beforeVc = beforeSearchComponent.makeView()
+    let headerFontSize: CGFloat = 16
+    let pointColor: UIColor = DesignSystemAsset.PrimaryColor.point.color
+    let grayColor: UIColor = DesignSystemAsset.GrayColor.gray400.color
+
+    lazy var beforeVc = beforeSearchComponent.makeView().then {
```

Comment:
```suggestion
    private lazy var beforeVc = beforeSearchComponent.makeView().then {
```
외부적으로는 안쓰이니 private 붙여주는게 좋을거같아요

Reply:
적용